src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -498,5 +500,65 @@ private static void ValidateRoleDefinition(PSRoleDefinition roleDefinition)
                 throw new ArgumentException(ProjectResources.InvalidActions);
             }
         }
+        public static void ValidateScope(string scope)
```

Comment:
>ValidateScope [](start = 27, length = 13)

one problem here is that if service changes, those changes won't reflect here. Ideally all such error messages maybe solved at service side.